plugins/node/instrumentation-mongoose/test/config.ts

```diff
@@ -18,3 +18,8 @@ export const MONGO_HOST = process.env.MONGODB_HOST || 'localhost';
 export const MONGO_PORT = Number(process.env.MONGODB_PORT || 27017);
 
 export const MONGO_URI = `mongodb://${MONGO_HOST}/${MONGO_PORT}`;
+
+export const shouldTest = process.env.RUN_MONGODB_TESTS != null;
+if (!shouldTest) {
+  console.log('Skipping mongodb tests. Set RUN_MONGODB_TESTS=1 to run them.');
```

Comment:
this is very helpful! 😄 
nit: I would use `true` instead of `1`, which seems more appropriate

Reply:
Though `1` was already being used in the CI workflow files:

```
.github/workflows/test-all-versions.yml
102:      RUN_MONGODB_TESTS: 1

.github/workflows/unit-test.yml
89:      RUN_MONGODB_TESTS: 1
```

I'm fine with using `true`, however.

Reply:
commit 503f867c